### DISCUSSION TITLE
Wet bias in cam6 (and cam_dev), potential issue with CLUBB to operate on dry density/mixing ratios

### DISCUSSION THREAD

**Discussion by tilmes** [NCAR/amwg_dev#300] [Tape recorder simulations]
Running one of the latest development tags for cam (cam6_3_112, and cam6_3_095) with both WACCM and CAMchem cam6 physics, we discovered a wet bias (high water vapor) in the stratosphere. Tracking down changes with cam6 (which are also in cam_dev) gives a suspicion that the wettening of the stratosphere may have been introduced by these changes:

**cam6_3_094: modify CLUBB to operate on dry density / mixing ratios** 

other changes going to cam6_3_095 include (but are not expected to change H2O much):
cam6_3_095: Chemistry updates   -- MAM5  +  NAT param and het chem updates (see https://github.com/ESCOMP/CAM/issues/558) 

I have performed a CAMchem (32L FV) simulation with tag cam6_3_93 (left panel) and compared to the same setup but with cam6_3_95 (right panel), you can see a significant increase in H2O in the stratosphere:

<img width="2202" alt="image" src="https://github.com/NCAR/amwg_dev/assets/22222754/392532c6-f5ad-4193-ab08-c3c29c2b6715">

We are also currently doing a WACCM 70L cam6 FV simulation, with cam6_3_93, that indicates that the wet bias detected using a later version of the code is removed. 

The change in CLUBB dry density may also have affected cam_dev code, resulting in too large H2O mixing ratios in the stratosphere.

**Comment by adamrher** [comment]
Hi Simone. If the right panel is 093, the left panel is 095, would you mind doing one more run with 094? I just want to be 100% sure its the 094 PR before I start diving into that PR.

And just for context, since we're looking L32 cam6, can you opine on how good/bad L58/L93 cam_dev UTLS moisture looks before and after the offending PR? How much more moist is it than the target (and I guess the target would be something like cam6_3_093 L32/L70 cam6, which is ~similar to cesm2 proper). How much did we over-correct for? Thanks!

**Reply by adamrher** [reply]
also some relevant context; we were tracking down cloud brightening in the 90s series tags and attributed it to 093 and 096 chem changes. That doesn't really explain the moistening between 093->095, but good to keep in mind. See Ben's table here: https://github.com/NCAR/amwg_dev/discussions/225#discussioncomment-5394007. And diagnostics here: https://github.com/NCAR/amwg_dev/discussions/225#discussioncomment-5455003

Maybe it would be worthwhile running the tape recorder script on all these 90s series runs from @bstephens82? I think they're still in his scratch archive directory.

**Reply by tilmes** [reply]
Hi Adam, I am wondering if we should just do some cam_dev L58 /L93 simulations using 093 and 094 to see if the same change in water vapor is occurring. For the recent L93 we clearly saw too much water vapor and @islasimpson was going to look at more comparisons to MLS this week.

The problem with the previous L58 / L93  run is that we included the H2O external forcings, which would make the runs wetter, to begin with, but it may still be interesting to see if we see the same changes. 

And yes, I can also do another 094 run if that would help.

**Reply by adamrher** [reply]
Ben's runs pre-date ghg_mam4 (which came in cam6_3_100) and therefore wouldn't have the H2O double counting issue. It might be wise to check these first before doing new sims. Maybe a first check is to see if his 093 and 095 tape recorders show the same sensitivity as your cam-chem/waccm runs you've plotted. He's running L58 cam_dev FHIST. If his runs do show the same behavior, then we can use his entire 90s series to get a handle on how the tape recorder has evolved in the 90 series tags, since that's where you've identified the problem occurs.

**Reply by islasimpson** [reply]
I've looked at #279 and #280  which are L58 and L32 runs.  They are definitely too wet.  At first, I thought they looked like the double counting issue had come back, but I don't think that's the case.  

Here is a plot...

![Screenshot from 2023-06-19 21-41-31](https://github.com/NCAR/amwg_dev/assets/41842751/2c7c8950-b55e-4a46-8971-6040eb256f2a)

and then one of the simulations with a more appropriate contour interval so you can compare a run from before the double counting issue was fixed and these current runs are.  Compare #279 and #269 for an L58 comparison.  We're clearly now a bit drier in #279 than in #269 in the upper stratosphere, which is presumably the effect of removing the double counting.  You can see the drying effect of the resolution change, but these are all too wet.

![Screenshot from 2023-06-19 21-44-22](https://github.com/NCAR/amwg_dev/assets/41842751/c61fe9c3-9493-48f9-a01f-c1d8f5e48389)

I'm having a hard time deciphering what other runs I should look at from the above.  If someone wants to send me a list, I can make a series of water vapor taper recorder plots for the required runs.

**Reply by JulioTBacmeister** [reply]
I think we should look at an L58 run before the "F-tuning" was implemented. I'll send some issue numbers soon.

**Reply by JulioTBacmeister** [reply]
Can we see what's happening in #235 and #229

**Reply by adamrher** [reply]
I am deeply skeptical about looking at tape recorder's in runs using FLTHIST before cam6_3_111, which is the first tag where the H2O double counting via the H20 external forcing file, was removed. I think the cleanest way forward is to look at Ben's **FHIST** runs that he used to produce this table for SWCF: https://github.com/NCAR/amwg_dev/discussions/225#discussioncomment-5394007. They also don't use F-tunings, and so it's a clean set of simulations.

**Reply by tilmes** [reply]
@adamrher My thought would be to just do a run that reverts the changes in cam3_093 (CLUBB changes) using the latest tag and then see where we are and then start from there to add the changes in again while analyzing the tape recorder (looking at H2O). I think, that may get us forward the quickest....?

**Reply by bstephens82** [reply]
My series of "default" runs using the 90s tags have similar paths, of this form:

/glade/scratch/stepheba/archive/dflt09X/atm/hist

where X goes from 1 to 9, and I also have dflt100.  

These are all 14-month F2000climo runs, cam_dev L58.

**Reply by JulioTBacmeister** [reply]
@adamrher I don't understand your skepticism.  I think looking at these runs will at least tell us whether it was the F-tuning that led to the moistening in the UTLS.

**Comment by islasimpson** [comment]
Runs #235 and #229 are definitely better.  You can see the issue of the double counting high up, but it's definitely clear in the lower stratosphere that they are better than the latest runs...
![Screenshot at 2023-06-20 09-04-08](https://github.com/NCAR/amwg_dev/assets/41842751/3715f011-34ec-46fd-adb3-a9c13f28a358)

**Reply by JulioTBacmeister** [reply]
Here is a slide from a CAM7 meeting back in early May
![image](https://github.com/NCAR/amwg_dev/assets/9221710/5c899c55-d438-450a-83ba-4a42e62b1f60)

I'd propose we do a few short sensitivity runs backing off the F-tuning:
F_1 = (  dcs:250 ,   **vtrmi:1.0** ,   wsubi:2.5 ,  seasalt:1.5. )
F_2 = (  dcs:250 ,   vtrmi:0.5 ,   **wsubi:1.0** ,  seasalt:1.5. )
F_3 = (  **dcs:500** ,   vtrmi:0.5 ,   wsubi:2.5 ,  seasalt:1.5. )

**Reply by tilmes** [reply]
@JulioTBacmeister can you explain how the F-tuning is related to adding CLUBB to operate on dry density/mixing ratios in 093.
I assume the F-tuning was done after 093, is that correct?

**Reply by bstephens82** [reply]
@tilmes I worked on the F-tuning around tag 099 or 100 or so, in an effort to get the SWCF, LWCF, and RESTOM back into more acceptable ranges.  I don't think of this F-tuning as being directly related to CLUBB.

**Reply by adamrher** [reply]
@tilmes while 094/095 did change the tape recorder (as you show), based in Isla's plots I think that it was something around 105 that caused things to get a lot wetter. Julio's suspects the F-tunings and so he has run-requests reverting those back right now (see under issues).

**Comment by islasimpson** [comment]
![h2o](https://github.com/NCAR/amwg_dev/assets/41842751/b77ec83e-2845-4d76-9514-8289611cca87)

**Reply by adamrher** [reply]
stating the obvious here. The UTLS moistening is due to the combination of reducing ```micro_mg_vtrmi_factor``` (1.0->0.5) and increasing ```microp_aero_wsubi_scale``` (1.0->2.5). Not sure what to do about this since we like what these two changes do to clouds.

We do have some wiggle room in that we need to dim the clouds by about 2 W/m2, when coupling to MOM6. But our problem there is mostly w/ low clouds, whereas these namelists seem to have a large effect on high clouds.

**Comment by islasimpson** [comment]
Here are some comparisons of the MT and LT runs.  First with the contour interval I've been using all along, second with a more appropriate contour interval.
![h2o_MT_vs_LT](https://github.com/NCAR/amwg_dev/assets/41842751/18d4dc21-d1bc-4ebe-bff4-f219f5feb14e)
![h2o_MT_vs_LT_appropriate_ci](https://github.com/NCAR/amwg_dev/assets/41842751/e6fc2167-0016-41f0-98a5-54b32d0a5ad9)

**Comment by islasimpson** [comment]
Here is also a comparison of the chem and no chem runs with MT.  At first glance it looks like a big difference around 30hPa, but I think it's actually because of the initialization.  The run isn't really long enough to say what's going on given the trajectory from lower values at the beginning of the run.
![chemistry_impact](https://github.com/NCAR/amwg_dev/assets/41842751/d956d21a-2b49-4167-addc-724351b1608a)

![Screenshot at 2023-06-23 16-44-11](https://github.com/NCAR/amwg_dev/assets/41842751/2ef2125f-041d-4390-8483-829ef41ed983)

**Reply by tilmes** [reply]
@islasimpson I have three years now. Do you want me to run longer?

**Reply by islasimpson** [reply]
Wasn't there a plan to run a longer chemistry run anyway to get the ozone field?  If so, we can just look at it in that run

**Reply by tilmes** [reply]
I think, we are doing  some GW tuning first, and then do a longer chemistry run.  However, I don't think, a new prescribed ozone field would make a difference in the wet bias...

**Reply by islasimpson** [reply]
I agree it may not make a difference.  My point was more that there's no need to run this one out longer if there are plans for another one that would be longer.  I don't think it's that urgent to answer this question and we can just look at it in the longer run when it's done.
